<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>鼻腔手術偵測與評價分析</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>50</y>
     <width>450</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>鼻腔手術中斷偵測與評價分析</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_choose_video">
   <property name="geometry">
    <rect>
     <x>482</x>
     <y>340</y>
     <width>236</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(100, 100,100);
border-radius: 20px;
border: 1px groove gray;
border-style:outset;</string>
   </property>
   <property name="text">
    <string>👆🏻 選擇資料夾 👆🏻</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_start_judge">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>482</x>
     <y>420</y>
     <width>236</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(100, 100, 100);
border-radius: 20px;
border: 1px groove gray;
border-style:outset;
</string>
   </property>
   <property name="text">
    <string>❗開始偵測❗</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_plot_result">
   <property name="geometry">
    <rect>
     <x>482</x>
     <y>500</y>
     <width>236</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(100, 100, 100);
border-radius: 20px;
border: 1px groove gray;
border-style:outset;
</string>
   </property>
   <property name="text">
    <string>顯示近五次結果圖 📊</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_manual">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>630</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(195, 195, 195);
border-radius: 50px;
border: 1px groove gray;
border-style:outset;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_video_name">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>200</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_folder_image">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>140</y>
     <width>170</width>
     <height>170</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_image">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>30</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_image_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_manual_2">
   <property name="geometry">
    <rect>
     <x>1128</x>
     <y>665</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        border: 0px solid;
    }</string>
   </property>
   <property name="text">
    <string>使用說明</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_check_folder">
   <property name="geometry">
    <rect>
     <x>482</x>
     <y>580</y>
     <width>236</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(100, 100,100);
border-radius: 20px;
border: 1px groove gray;
border-style:outset;</string>
   </property>
   <property name="text">
    <string> 查看已偵測過資料夾 📝</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_back_menu">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>600</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        border: 0px solid;
    }
    QPushButton:hover {
        background-color: rgb(223, 223, 223);
    }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/back.png</normaloff>image/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="list_widget_done_folder">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>160</y>
     <width>370</width>
     <height>510</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>label_folder_image</zorder>
  <zorder>label_title</zorder>
  <zorder>button_choose_video</zorder>
  <zorder>button_start_judge</zorder>
  <zorder>button_plot_result</zorder>
  <zorder>button_manual</zorder>
  <zorder>label_video_name</zorder>
  <zorder>label_title_image</zorder>
  <zorder>label_title_image_2</zorder>
  <zorder>button_manual_2</zorder>
  <zorder>button_check_folder</zorder>
  <zorder>button_back_menu</zorder>
  <zorder>list_widget_done_folder</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
